examples/webpack/src/index.js

```diff
@@ -14,14 +14,12 @@ const authSuccess = ({ authResponse }, parallel) => {
 
   document.getElementById('authResponse').innerHTML = JSON.stringify(authResponse)
 
-  if (!scopes.includes('accreditation_status')) return
-
-  parallel.api('/accreditations', ({ accreditations }) => {
-    if (accreditations.some(({ status }) => status === 'current')) {
-      msg.innerHTML += ' Also you are accredited!  Congrats!'
-    } else {
-      msg.innerHTML += ' Also, looks like you are not accredited. :('
-    }
+  parallel.api('/profile', (profile) => {
```

Comment:
Think we should remove all calls/references to the `api()` function:

```suggestion
  parallel.getProfile((profile) => {
```